Dockerfile

```diff
@@ -1,8 +1,6 @@
-FROM jalle19/centos7-stlinux24:latest
```

Comment:
if we remove this, 1.3.X will not be able to compile AXE target, so maybe we should leave it for a while.

Reply:
We should publish a `1.3.x` tag using the current `master` in this repository use that image when building 1.3.x

Reply:
You are right let’s do that

Reply:
@catalinii you're gonna have to do it, it's your Docker Hub token. After that we can update the `1.3`  branch in minisatip to point to the new tag and merge this one and start iterating.

Reply:
Done, created a new tag for minisatip 1.3 and switched minisatip/1.3 to that tag